analyzers/HIBP_Query/HIBP_Query.json

```
@@ -0,0 +1,15 @@
+{
+    "name": "HIBP_Query",
+    "version": "1.0",
+    "author": "Matt Erasmus",
+    "url": "https://github.com/TheHive-Project/Cortex-Analyzers",
+    "license": "AGPL-V3",
+    "description": "Query haveibeenpwned.com for a compromised email address",
+    "dataTypeList": ["mail"],
+    "baseConfig": "HIBP_Query",
+    "config": {
+      "service": "query",
+      "url": "https://haveibeenpwned.com/api/v2/breachedaccount/"
+    },
+    "command": "HIBP_Query/hibpquery_analyzer.py"
```

```suggestion
    "command": "HIBP_Query/hibpquery_analyzer.py",
    "configurationItems": [
    {
      "name": "unverified",
      "description": "Include unverified breaches",
      "type": "boolean",
      "multi": false,
      "required": true,
      "defaultValue": true
    }
    ]
```
I think it would be beneficial to make available the option to include "unverified" breaches. [Searching All Breaches via API](https://haveibeenpwned.com/API/v2#BreachesForAccount)